### PR TITLE
refactor: remove test value

### PR DESCRIPTION
**Pull request by r3v4s**
## refactor
1. removed value for testing with testcase and on-chain tests.
> each testcase overwrites from testing logic
> on-chain test value will be overwrite when we actually deploy those.

### CODE REVIEW

**Review by notJoon — DISMISSED** [review]

File: gov/governance/config.gno [hunk 1]
```diff
@@ -17,40 +17,17 @@ var (
...
-		// XXX: TODO, REMOVE AFTER QA
-		// - config for live qa
-		// VotingStartDelay:              uint64(180),           // 3m
-		// VotingPeriod:                  uint64(600),           // 10m
-		// VotingWeightSmoothingDuration: uint64(600),           // 10m
-		// Quorum:                        uint64(50),            // 50% of total xGNS supply
-		// ProposalCreationThreshold:     uint64(1_000_000_000), // 1_000_000_000
-		// ExecutionDelay:                uint64(60),            // 1m
-		// ExecutionWindow:               uint64(3600),          // 1h
+		VotingStartDelay:              uint64(86400),         // 1d
```
Comment (onlyhyde) [inline gov/governance/config.gno:22]: as i know, dongwon have request to use governanve config for QA. need to check it.

**Review by onlyhyde — CHANGES_REQUESTED** [review]

File: gov/governance/config.gno (hunk 1 above)
Comment (dongwon8247) [reply gov/governance/config.gno:22]: yes, let's set all configs with QA time for now, and when we're done with the final QA before `beta v3`, we can roll all back.

Comment (r3v4s) [reply gov/governance/config.gno:22]: FYI, current dev chain deployed with qa timing

Comment (r3v4s) [reply gov/governance/config.gno]: 9bb9416667c213fff1860a24ff3686b9aaef4c69

**Review by onlyhyde — APPROVED** [review]

**Review by onlyhyde — DISMISSED** [review]

**Review by notJoon — APPROVED** [review]

**Review by onlyhyde — APPROVED** [review]

**Pull request merged**